internal/common/objects/BaseBiker.go

```diff
@@ -165,12 +165,11 @@ func (bb *BaseBiker) DecideForce(direction uuid.UUID) {
 
```

Comment:
Could you just update the function description from 
`// the function is passed in the id of the voted lootbox, for now ignored` to `// the function is passed in the id of the voted lootbox and the default base bikers steer to that lootbox.`